README.md

```diff
@@ -1,19 +1,107 @@
 # śiva format शिव for the JVM [![Build Status](https://travis-ci.org/src-d/siva-java.svg?branch=master)](https://travis-ci.org/src-d/siva-java)
 
 This library is a Java implementation of [siva format](https://github.com/src-d/go-siva/blob/master/SPEC.md).
-It  is intended to be used with any JVM language. 
+It  is intended to be used with any JVM language.
 The main implementation is written in Go [here](https://github.com/src-d/go-siva).
 
-## Tests
+This java library offers an API to read and unpack [siva files](https://github.com/src-d/go-siva/blob/master/SPEC.md) but not to write them yet.
 
-Just run `sbt test`
+## Usage
 
-## Installation
+For the moment `siva-java` is available on [jitpack](https://jitpack.io/#src-d/siva-java/). To include it as a dependency in your project managed by [sbt](http://www.scala-sbt.org/), add the resolver and the dependency to your `build.sbt` file:
```

Comment:
Now is in maven central, right?

Reply:
Now it's on maven

Reply:
I can't find [siva-java on maven](https://search.maven.org/#search%7Cga%7C1%7Csiva-java)